templates/interface/filter-panels.interface.hbs

```diff
@@ -58,9 +58,27 @@
         </div>
       </div>
 
+      <div class="form-group clearfix" id="filter-value-type">
+        <div class="col-sm-4 control-label">
+          <label>Value type</label>
+        </div>
+        <div class="col-sm-8">
+          <label for="value-type-field-{{id}}" class="select-proxy-display">
+            <select name="value-type-field-{{id}}" id="value-type-field-{{id}}" data-field="valueType" data-label="-- Enter a value" class="hidden-select form-control">
+              <option value="enter-value">Enter a value</option>
+              <option value="user-profile-data">User profile data</option>
+              <option value="app-storage-variable">App storage variable</option>
+              <option value="link-query-parameter">Link query parameter</option>
+            </select>
+            <span class="icon fa fa-chevron-down"></span>
+            <span class="select-value-proxy">-- Please wait</span>
```

Comment:
I don't think we need this because:

1. There is no loading state for this dropdown because there's no asynchronous process before showing this
1. A `.select-value-proxy` container isn't necessary for dropdowns. See https://developers.fliplet.com/UI-guidelines-interface.html#dropdown-list

```suggestion
```

Reply:
removed